src/domain/users/user-organizations.repository.integration.spec.ts

```diff
@@ -1430,27 +1430,29 @@ describe('UserOrganizationsRepository', () => {
       ).rejects.toThrow('User not found.');
     });
 
-    it('should throw an error if the org does not exist', async () => {
+    it('should throw an error if the user is not a member of the organization', async () => {
```

Comment:
As the membership check (i.e., whether a `UserOrganization` for the `userId` and `organizationId` pair exists) is done before getting the organization data, checking the organization existence does not make sense anymore. It should exist if a `UserOrganization` for the organization exists.